modules/dnn/src/dnn.cpp

```diff
@@ -1670,26 +1831,5 @@ Ptr<Layer> LayerFactory::createLayerInstance(const String &_type, LayerParams& p
     }
 }
 
-BackendNode::BackendNode(int backendId) : backendId(backendId) {}
```

Comment:
These classes are base classes for `HalideBackendNode` and `HalideBackendWrapper`. If Halide backend enabled, implementations undefined.